src/thing_creature.c

```diff
@@ -2591,6 +2591,15 @@ TbBool kill_creature(struct Thing *creatng, struct Thing *killertng,
     SYNCDBG(18,"Starting");
     TRACE_THING(creatng);
     //return _DK_kill_creature(creatng, killertng, killer_plyr_idx, (flags&CrDed_NoEffects)!=0, (flags&CrDed_DiedInBattle)!=0, (flags&CrDed_NoUnconscious)!=0);
```

Comment:
for old functions like this the dll reference should probably be removed if you're working on the function